tests/tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/jenkinsfile-runner-test-framework"
+sh_unit_directory="$test_framework_directory/shunit2"
+
+version="256.0-test"
+CWP_version="1.3"
```

Comment:
Why do you use old version here? 1.5 is the current version

Reply:
Ideally it should be using a default version set in the JFR Test Framework, just to K.I.S.S

Reply:
It was just an initial example, but I will change it.
Regarding the default version, I'll see how to add it, but let's do it in a follow-up